### DISCUSSION TITLE
Disable 2FA from database

### DISCUSSION THREAD

**Discussion by alexynadamas** [YetiForceCompany/YetiForceCRM#15037] [How to]
Hi

I have a problem with an admin account with his 2FA, then I can't log in. I want to disable 2FA out of the backend.

Could I disable it from DB? What is the table name?

thanks

**Comment by mariuszkrzaczkowski** [comment]
Disable 2FA or LDAP auth of any user script

https://github.com/YetiForceCompany/YetiForceScripts/tree/master/Disable2FAorLDAP

__CLI__ in CRM https://github.com/YetiForceCompany/YetiForceCRM/commit/331f12dc8291272f1056f4fbd1d145bbd59ad354

```bash
php cli.php -m Users -a passwordAuth -l admin
```

**Reply by alexynadamas** [reply]
Thanks

Is good idea, as an alternative, rename PLL_PASSWORD_FA field to PLL_PASSWORD on table vtiger_users?

**Reply by mariuszkrzaczkowski** [reply]
it should work, but may need to refresh the permissions files

**Reply by alexynadamas** [reply]
What do you mean?
